kibana/install.sh

```diff
@@ -6,10 +6,48 @@ set -o pipefail
 
 source "${HOME}/prep-install.${RELEASE_STREAM}"
 
+# https://stackoverflow.com/questions/4023830/how-to-compare-two-strings-in-dot-separated-version-format-in-bash
+compare_versions() {
+    if [[ $1 == $2 ]]
+    then
+        return 0
+    fi
+    local IFS=.
+    local i ver1=($1) ver2=($2)
+    # fill empty fields in ver1 with zeros
+    for ((i=${#ver1[@]}; i<${#ver2[@]}; i++))
+    do
+        ver1[i]=0
+    done
+    for ((i=0; i<${#ver1[@]}; i++))
+    do
+        if [[ -z ${ver2[i]} ]]
+        then
+            # fill empty fields in ver2 with zeros
+            ver2[i]=0
+        fi
+        if ((10#${ver1[i]} > 10#${ver2[i]}))
+        then
+            return 1
+        fi
+        if ((10#${ver1[i]} < 10#${ver2[i]}))
+        then
+            return 2
+        fi
+    done
+    return 0
+}
+
+if compare_versions "$KIBANA_VER" "5.6.13" "=" ; then
```

Comment:
The `"="` argument is unused?
The problem is only in 5.6.13?

Reply:
Updated.  its a 5.6.x issue.. that was fixed in 6.x.  My concern was a version difference in the library but i changed it to be 5.6.13+